src/test/java/AndreeaG/tests/tema5/Demoqa/Buttons.java

```diff
@@ -0,0 +1,36 @@
+package AndreeaG.tests.tema5.Demoqa;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import teofilursan.driver.BrowserManager;
+
+public class Buttons {
+
+
+    public static void main(String[] args) {
+     clickButtons();
+
```

Comment:
No need to leave one blank line before the closing '}' at the end of methods. You can remove that line. Check on other methods from all the classes too. Thanks!

Reply:
I will clean them all up! thank you!